sample-graphs/update.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -eu
+
+sourcecred_module="$(dirname "$0")""/../sourcecred"
```

Comment:
The close-quote open-quote in the middle is superfluous; it is
equivalent (and more legible) to write

```shell
sourcecred_module="$(dirname "$0")/../sourcecred"
```

Reply:
Thanks. I'm still a little shaky on what kind of expansion happens inside double quotes, which leads me to producing occasionally odd code.